roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,43 @@
+# pylint: disable=missing-docstring
+
+"""Interface to OpenShift oc command"""
+
+import shlex
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            namespace=dict(type="str", required=True),
+            config_file=dict(type="str", required=True),
+            cmd=dict(type="str", required=True)
+        ),
+    )
+
+    cmd = ['oc',
```

Comment:
Thanks, tried using `openshift.common.client_binary` but, at least in my case, it just returned `oc` as its value, no full path. Copied the `locate_oc_binary` func from `base.py`